modules/stanford_faq_related_show_hide_view/stanford_faq_related_show_hide_view.info

```diff
@@ -2,9 +2,11 @@ name = Stanford FAQ Related Show-Hide View
 description = FAQs view with bootstrap collapse (show/hide) classes. Displays FAQs related to the page.
 core = 7.x
 package = Stanford
-version = 7.x-4.x-dev
+version = 7.x-4.0-dev
```

Comment:
These versions should be 7.x-4.3-dev. I know the other versions are in the format `7.x-4.x-dev` but we were young and foolish at the time.

Reply:
@cjwest Please make all the versions be `7.x-4.3-dev` throughout